src/components/Sidebars/Generate.js

```diff
@@ -99,22 +46,13 @@ const Generate = ({ cdpId, reset }) => {
           type="number"
           value={amount}
           min="0"
-          onChange={onAmountChange}
+          onChange={({ target }) => setAmount(target.value)}
           placeholder="0.00 DAI"
-          failureMessage={amountErrors}
-        />
-        <RatioDisplay
-          type={RatioDisplayTypes.CARD}
-          ratio={collateralizationRatio}
-          ilkLiqRatio={liquidationRatio}
-          text={lang.action_sidebar.generate_warning}
-          onlyWarnings={true}
-          show={amount !== '' && amount > 0 && !undercollateralized}
-          textAlign="center"
```

Comment:
These warnings were part of CDP-533? Is there any reason for them to be removed?

Reply:
nah, just got lost in a rebase. plan to go through and merge them back manually